recipes/openvdb/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "openvdb" %}
+{% set version = "10.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AcademySoftwareFoundation/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 887a3391fbd96b20c77914f4fb3ab4b33d26e5fc479aa036d395def5523c622f
+
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+
+  host:
+    - boost-cpp
+    - blosc
+    - zlib
+    - tbb-devel
+    - jemalloc
+
+  run:
+    - boost-cpp
+    - blosc
+    - zlib
+    - tbb-devel
+    - jemalloc
```

```suggestion
  run:
    - boost-cpp
```
According to the logs, you are overdepending on jemalloc, zlib, and tbb-devel. If these are header-only libraries, you don't need to list them in the run requirements. If you are static linking, don't! static linking is not allowed.

Generally recipes should have run_exports, so you should not need to list C libraries in run requirements explicitly unless conda-build complains that something is missing.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=745563&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1493

---

Fixed in https://github.com/versatran01/staged-recipes/pull/1/commits/646e474ffa3b84ba8bfd4256495f38db99f6f88e.